docs/runbooks/VirtHandlerRESTErrorsBurst.md

```diff
@@ -1,17 +1,55 @@
+<!-- Edited by Jiří Herrmann, 7 Nov 2022 -->
+
 # VirtHandlerRESTErrorsBurst
 
 ## Meaning
 
-More than 80% of the rest calls failed in virt-handler for the last 5 minutes
+More than 80% of REST calls failed in `virt-handler` in the last 5 minutes. This alert usually indicates that the `virt-handler` pods cannot connect to the API server.
+
+This error is most frequently caused by one of the following problems:
+
+- The API server is overloaded, which causes timeouts. To verify if this is the case, check the metrics of the API server, and view its response times and overall calls.
+
+- The `virt-handler` pod cannot reach the API server. This is commonly caused by DNS issues on the node and networking connectivity issues.
 
-virt-handler has potentially fully lost connectivity to the apiserver. Running workloads on the affected node will not be impacted but status updates will not be propagated and actions like migrations can not take place.
+## Impact
+
+Running workloads on the affected node are not impacted, but status updates are not propagated and node-related actions, such as migrations, fail.
 
 ## Diagnosis
 
-There exist two common types of errors:
-- The apiserver is overloaded and we run into timeouts. Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls, …
-- The virt-handler pod can’t reach the apiserver. Common issues are: dns issues on the node, networking connectivity issues
+Check whether `virt-handler` can connect to the API server.
+
+1. Set the `NAMESPACE` environment variable:
+    ```
+     $ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+    ```
+
+2. Obtain the name of the `virt-handler` pod:
```

Comment:
"Obtain or check the status of the ... pod:" because the command gets more than just the pod name.